pkgs/tools/filesystems/ceph/default.nix

```diff
@@ -318,6 +318,11 @@ in rec {
     pname = "ceph";
     inherit src version;
 
+    postPatch = ''
+      substituteInPlace cmake/modules/Finduring.cmake \
+        --replace-fail "liburing.a liburing" "uring"
```

Comment:
Fix is in upstream Ceph for Ceph 19 (unreleased): https://github.com/ceph/ceph/commit/670118abfb9ef6135972a56514173e72b64ef8f4